plan/plans.go

```diff
@@ -169,11 +170,12 @@ type Simple struct {
 type Insert struct {
 	baseLogicalPlan
 
-	Table       *ast.TableRefsClause
+	Table       *model.TableInfo
```

Comment:
Use `table.Table` instead of `*model.TableInfo`, 
`Table.Cols` returns public columns only.